packages/dds/tree/src/util/rangeMap.ts

```diff
@@ -92,6 +92,33 @@ export class RangeMap<K, V> {
 		return entries;
 	}
 
+	// XXX: Merge with getAll
+	/**
+	 * Like getAll, but includes entries where the value is undefined.
+	 */
+	public getAll2(start: K, length: number): RangeQueryResult<K, V>[] {
```

Comment:
Would be good to add test coverage for this. I'm happy to take that on to save you some time.